testing/cloudbuild-templates/cloud-run-template.cloudbuild.yaml

```diff
@@ -17,33 +17,39 @@ steps:
   - | # Cloud Run Service name must be less than 63 characters
     cat /dev/urandom | LC_CTYPE=C tr -dc 'a-z0-9' | head -c 15 > _short_id # Generate a random 15 character alphanumeric string (lowercase only)
     gcloud run deploy ${_SERVICE}-$(cat _short_id) \
-     --image gcr.io/${PROJECT_ID}/${_SERVICE}:${SHORT_SHA} \
-     --no-allow-unauthenticated \
-     --region ${_SERVICE_REGION} \
```

Comment:
`--region` now controlled via environment variable in `options.env`. Flipping the substitutions to environment variables or arguments is necessary to support the separated common.sh script.